install.md

```diff
@@ -0,0 +1,108 @@
+<!-- vim-markdown-toc GFM -->
+
+* [Nmstate Installation Guide](#nmstate-installation-guide)
+    * [RPM based](#rpm-based)
+        * [Stable release](#stable-release)
+        * [Developer Branch](#developer-branch)
+    * [PyPI/pip](#pypipip)
+        * [Stable Release](#stable-release-1)
+        * [Developer Branch](#developer-branch-1)
+    * [setup.py](#setuppy)
+        * [Stable Release](#stable-release-2)
+        * [Developer Branch](#developer-branch-2)
+
+<!-- vim-markdown-toc -->
+
+# Nmstate Installation Guide
```

Comment:
IMHO it should also mention that NetworkManager is usually required and NetworkManager-ovs to manager OVS and that NM needs to be restarted after installinng NetworkManager-ovs and openvswitch to get it to work.

Reply:
Can we just raise and exception warning user `NetworkManager-ovs` or `openvswitch` not installed?

And restart `NetworkManager.service` should be done by `NetworkManager-ovs` rpm, not our document.

I don't like this non-nmstate things in installation guide.

Reply:
IIRC there is no specific failure for missing OVS support except that the desired state will not be reached. Ideally NetworkManager-ovs would take care of restarting but since it is not, it is important for users to know this.

Reply:
Add them into `Runtime Dependency` section. And hopefully someday we could remove them after we raise proper exception for missing runtime dependency.